README.md

```diff
@@ -330,120 +477,69 @@ Additionally, you can also locally trigger a deep link to make sure your code is
 
-At this point unfortunately we don't support testing debug builds with Klaviyo. So if you are trying to send a test push notification to a debug build you'll see an error on Klaviyo.
-
-A suggested temporary workaround would be creating a test flight build with the above changes required for rich push notifications, performing some actions on the test flight build to identify the device and making sure you are able to see that device in Klaviyo. Once you have that device's push token in any profile you can create a list or segment with that profile and send a push campaign with an image to test the full end-to-end integration.
-
-## SDK Data Transfer
-
-Starting with version 1.7.0, the SDK will cache incoming data and flush it back to the Klaviyo API on an interval. The interval is based on the network link currently in use by the app. The table below shows the flush interval used for each type of connection:
+### SDK Data Transfer
+Starting with version 1.7.0, the SDK will cache incoming data and flush it back to the Klaviyo API on an interval.
+The interval is based on the network link currently in use by the app. The table below shows the flush interval used for each type of connection:
 
 | Network   | Interval   |
 | :-------- | :--------- |
 | WWAN/Wifi | 10 seconds |
 | Cellular  | 30 seconds |
 
-Connection determination is based on notifications from our reachability service. When there is no network available, the SDK will cache data until the network becomes available again. All data sent by the SDK should be available shortly after it is flushed by the SDK.
+Connection determination is based on notifications from our reachability service.
+When there is no network available, the SDK will cache data until the network becomes available again.
+All data sent by the SDK should be available shortly after it is flushed by the SDK.
 
 ### Retries
+The SDK will retry API requests that fail under certain conditions. For example, if a network timeout occurs, the request will be retried on the next flush interval.
+In addition, if the SDK receives a rate limiting error `429` from the Klaviyo API, it will use exponential backoff with jitter to retry the next request.
 
-The SDK will retry API requests that fail under certain conditions. For example, if a network timeout occurs, the request will be retried on the next flush interval. In addition, if the SDK receives a rate limiting error `429` from the Klaviyo API, it will use exponential backoff with jitter to retry the next request.
-
-## License
-
+### License
 KlaviyoSwift is available under the MIT license. See the LICENSE file for more info.
 
+### UserDefaults access (SDK version < 3.0.0)
+As of fall 2023, Apple requires apps that access specific Apple APIs to provide a reason for this access.
+Previous versions of the Klaviyo SDK used UserDefaults to store data about the current user.
+Today, when the SDK starts up, it must access this data to migrate to a new format.
```

Comment:
Should we change `today` to say in versions below 3.0.0 or something...

Reply:
Hmm good call. Can we get rid of this section now that we don't use user defaults? Yeah some of our older versions do but we've never heard anyone ask about this so far. CC: @mark-piana

Reply:
You're probably right. I'm not sure why we'd need to include this anymore since anyone just starting to implement things wouldn't care about this information.

Reply:
Yeah IDK how important this whole section is, maybe it could be called out in the migration guide or something, but doesn't feel like its relevant in the README for v3.0+

Reply:
I'll just remove it for now. I don't think Apple is strict about enforcing this either.